src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -378,6 +378,19 @@ struct GhInterfaceManager : db::SimpleTag {
   }
 };
 
+struct InterfaceManagerInterpolationStrategy : db::SimpleTag {
+  using type = InterfaceManagers::InterpolationStrategy;
+  using option_tags = tmpl::list<OptionTags::GhInterfaceManager>;
+
+  static constexpr bool pass_metavariables = false;
+  static InterfaceManagers::InterpolationStrategy
+  create_from_options(
```

Comment:
We are confused about who choose the strategy...  It looks like it should be the InterfaceManager, but here you are allowing it to be option-creatable ...

Reply:
Yeah, I can see how this logic is a little confusing.
The reason I put it together this way is that the interpolation strategy needs to be in the const global cache, but it is determined by which interface manager is chosen by the options, and the interface manager does not end up in the const global cache because it stores stateful data that should be held only by the worldtube component. 

So my solution was to have this option creatable, but reference the same option tag used to make the interface manager, so that it can get the appropriate interpolation strategy when the const global cache is initialized from options.
But, I admit that this is a bit confusing, so I'm happy to implement any alternatives that seem less tangled.